app.py

```diff
@@ -40,6 +43,9 @@ def handle_error(e):
 # app.config['JWT_BLACKLIST_ENABLED'] = True
 app.config['JWT_BLACKLIST_TOKEN_CHECKS'] = ['access']
 
+logging.basicConfig(filename='app.log', filemode='w', format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+# logging.warning('This will get logged to a file')
+
 from endpoints.manga.controller import UserMangaResource, UserMangaByIdResource
```

Comment:
Why are you importing this afterwards?

Reply:
Can't actually import them before "db = SQLAlchemy(app)" as the models depend on it.